### PR TITLE
fixes timezone error

### PR DESCRIPTION
**Pull request by beydogan**

### CODE REVIEW

File: roles/postgres/templates/postgresql.conf.j2 [hunk 1]
```diff
@@ -56,7 +56,7 @@ external_pid_file = '/var/run/postgresql/9.3-main.pid'      # write an extra PID
 
 # - Connection Settings -
 
-listen_addresses = 'localhost, 10.11.12.13'     # what IP address(es) to listen on;
+listen_addresses = '*'     # what IP address(es) to listen on;
```
Comment (phawk) [inline roles/postgres/templates/postgresql.conf.j2:59]: Would prefer a secure default here. Digital ocean and the likes offer private networking and it’s much better to only bind to the private network address here meaning your server can’t be accessed outside of the digital ocean stack.

File: roles/ruby/tasks/ruby21source.yml [hunk 2]
```diff
@@ -53,4 +53,4 @@
   command: make install chdir={{tmpRubyPath}} creates=/usr/local/bin/ruby
 
 - name: Install Bundler
-  gem: name=bundler state=latest
+  command: gem install bundler
```
Comment (phawk) [inline roles/ruby/tasks/ruby21source.yml:56]: Why have you changed this from using ansibles built in gem handler?

Comment (beydogan) [inline roles/ruby/tasks/ruby21source.yml:56]: Cant remember exactly but I think it was giving an error. I had to change.

**Comment by phawk** [comment]
Sorry for missing your pull request! Need to sort my GH notification emails out.

Firstly apologies for this not being actively maintained. I pushed it up to GH on request from a comment on reddit. I maintain this offline in a private repo and have implemented my own bootstrap.yml file. I think you’d be best maintaining this on your own if you want to actively look at it. I can update the readme to point people to your version if they are actively looking to use it.

**Comment by beydogan** [comment]
No need to apology :) Actually I've sent you the pull request for only one commit to fix timezone error. Because it wasn't running without it. I haven't sent pull request for all of my commits.

**Pull request closed without merging**